docs/development-setup.md

```diff
@@ -57,6 +57,14 @@ adb reverse tcp:8000 tcp:8000
 12. Take a look at the style by viewing the test map: [http://localhost:5002](http://localhost:5002)
 13. Take a look at the backend: [http://localhost:8000](http://localhost:8000) (The public version is available at
     api.entitlementcard.app)
+14. Set up the matomo instance [http://localhost:5003](http://localhost:5003) (The public version is available at https://matomo-entitlementcard.tuerantuer.org)
+15. (optional) Add your matomo config to the backend config
+```yaml
+matomo:
```

Comment:
shouldn't this be added to config-local.yaml?
Even i do that i can not start the backend since
```js
Exception in thread "main" com.fasterxml.jackson.databind.exc.UnrecognizedPropertyException: Unrecognized field "matomo" (class app.ehrenamtskarte.backend.config.BackendConfiguration), not marked as ignorable (7 known properties: "server", "production", "projects", "geocoding", "map", "postgres", "csvWriter"])
```
Where exactly do i have to add this @sarahsporck

Reply:
you need to add this to the project config. that's in the config.local.yml e.g.
```yaml
projects:
  - id: nuernberg.sozialpass.app
  ...
  matomo:
    ...
```

Reply:
so not in the first level? probably this was my problem. i got an error when i started the backend with this additional parameters

Reply:
could you please update the docs that its easier to know where to add matomo